lib/host/wasi_nn/CMakeLists.txt

```diff
@@ -5,8 +5,33 @@ wasmedge_add_library(wasmedgeHostModuleWasiNN
   wasinnmodule.cpp
 )
 
-target_link_libraries(wasmedgeHostModuleWasiNN
+list(APPEND INCLUDE_DIRS "${Boost_INCLUDE_DIRS}")
+list(APPEND INCLUDE_DIRS "${PROJECT_SOURCE_DIR}/include")
+list(APPEND INCLUDES_DIRS "${PROJECT_SOURCE_DIR}/thirdparty")
+
+
+list(APPEND LIB_DIRS "wasmedgeSystem")
+
+# add backends building flags
+add_definitions(-DWASINN_BUILD_OPENVINO)
+
+
+get_directory_property(WASINNDEFS COMPILE_DEFINITIONS)
+if(WASINNDEFS MATCHES "WASINN_BUILD_OPENVINO")
```

Comment:
You can refer to the same coding style of the flags such as `WASMEDGE_BUILD_AOT_RUNTIME` or others.